alexaapi/request.go

```diff
@@ -51,6 +51,7 @@ const (
 	RequestTypeIntentRequest           RequestType = "IntentRequest"
 	RequestTypeSessionEndedRequest     RequestType = "SessionEndedRequest"
 	RequestTypeSessionResumedRequest   RequestType = "SessionResumedRequest"
+	RequestTypeAudioPlayer             RequestType = "AudioPlayer"
```

Comment:
Need to verify whether or not this is an actual request defined by the Alexa API

Reply:
It is, here's a sample
```
{
    "context": {
        "AudioPlayer": {
            "offsetInMilliseconds": 13002,
            "playerActivity": "STOPPED",
            "token": "amzn1.echo-api.session.c39d904b-ac2f-417e-806d-a3581b4c1973:audio"
        },
        "System": ...
        "Viewport": ....
   }
}
```